npm/handler.go

```diff
@@ -93,11 +95,18 @@ func (m *NPM) SetRootModule(path string) error {
 
 // GetRootModule return root package information ex. Name, Version
 func (m *NPM) GetRootModule(path string) (*meta.Package, error) {
-	r := reader.New(filepath.Join(path, m.metadata.Manifest[0]))
-	pkResult, err := r.ReadJSON()
+	// Join the given path with the name of the manifest file.
+	p := filepath.Join(path, m.metadata.Manifest[0])
+	// Read the contents of the manifest file.
+	pBytes, err := os.ReadFile(p)
 	if err != nil {
 		return nil, err
 	}
+	// Unmarshal the JSON data from the manifest file into a map.
+	pkResult := make(map[string]interface{})
+	if err := json.Unmarshal(pBytes, &pkResult); err != nil {
+		return nil, err
```

Comment:
Please wrap your errors to add context on where we are failing
```suggestion
		return nil, fmt.Errorf("unmarshalling manifest: %w", err)
```